runtime/compiler/runtime/RelocationRuntime.hpp

```diff
@@ -263,7 +263,8 @@ class TR_RelocationRuntime {
       void relocateAOTCodeAndData(U_8 *tempDataStart,
                                   U_8 *oldDataStart,
                                   U_8 *codeStart,
-                                  U_8 *oldCodeStart);
+                                  U_8 *oldCodeStart,
+                                  uint8_t *existingCode = NULL);
```

Comment:
Did you intend to use this `existingCode` parameter? It doesn't look like it has a purpose in this patch.

Reply:
I do use it here:
https://github.com/eclipse/openj9/blob/54d3d6215b0202d1dc4f8f5f30a250416a8b46a8/runtime/compiler/runtime/RelocationRuntime.cpp#L457

Reply:
Right, but looking at the code I don't see the point where `relocateAOTCodeAndData` is invoked with `existingCode`.

Reply:
Right, I forgot to add it.